assets/stylesheets/_main.scss

```diff
@@ -2,6 +2,13 @@
  * Layout Views
  */
 
+// DO NOT MERGE THIS
+html,
+body,
+.wpcom-site {
+	height: 100%;
+}
+
```

Comment:
Actually, I'm thinking of including it as it makes working with layout easier :) But I need to edit the comment.